### PR TITLE
Add r-ggjoy.

### PR DESCRIPTION
**Pull request by cbrueffer**

### CODE REVIEW

**Comment by jenzopr** [comment]
Just FYI, `ggjoy` is deprecated by the upstream author[1]. The follow up is called `ggrides`[2]. I don't know if this has to be handled in a special way then?

[1] https://github.com/clauswilke/ggjoy
[2] https://github.com/clauswilke/ggridges

**Comment by cbrueffer** [comment]
Indeed; this ```ggjoy``` version actually depends on ```ggridges```. I still think it's useful for backwards compatibility.

File: recipes/r-ggjoy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - r-ggplot2 >=2.2.0
+    - r-ggridges >=0.4.0
+
+test:
+  commands:
+    - $R -e "library('ggjoy')"  # [not win]
+    - "\"%R%\" -e \"library('ggjoy')\""  # [win]
+
+about:
+  home: https://cran.r-project.org/web/packages/ggjoy/index.html
```
Comment (jdblischak) [inline recipes/r-ggjoy/meta.yaml]: Not a huge deal, but when you have to manually fix the home URL, it's best to use the URL listed on the CRAN page (which is what the cran skeleton normally does when working correctly). In this case that would have been https://github.com/clauswilke/ggjoy

**Comment by cbrueffer** [comment]
Fixed, thanks for the review John!

**Pull request merged**